valhalla/worker.h

```diff
@@ -1,5 +1,6 @@
-#ifndef __VALHALLA_SERVICE_H__
-#define __VALHALLA_SERVICE_H__
+#ifndef __VALHALLA_WORKER_H__
```

Comment:
had the wrong include guard. not yet a problem since valhalla_service doesn't have a header